info-compiler/src/test/java/com/google/sps/InfoCompilerTest.java

```diff
@@ -119,13 +123,30 @@ public static void initialize() throws InterruptedException, IOException {
     channels.add(Twitterchannel);
     channels.add(facebookChannel);
     candidate.add("channels", channels);
-
     JsonArray candidates = new JsonArray();
     candidates.add(candidate);
     singleContestJson = new JsonObject();
     singleContestJson.addProperty("office", "Governor");
     singleContestJson.add("candidates", candidates);
-  }
+    
+    JsonObject official = new JsonObject();
+    official.addProperty("name", "Andrew Cuomo");
+    JsonArray officials = new JsonArray();
+    officials.add(official);
+    official = new JsonObject();
+    official.addProperty("name", "John Doe");
+    officials.add(official);
+    JsonObject office = new JsonObject();
+    office.addProperty("name", "Governor of New York");
+    JsonArray officialIndices = new JsonArray();
+    officialIndices.add(0);
+    office.add("officialIndices", officialIndices);
+    JsonArray offices = new JsonArray();
+    offices.add(office);
+    representativesJson = new JsonObject();
+    representativesJson.add("offices", offices);
+    representativesJson.add("officials", officials);
```

Comment:
If it's easier you can probably just implement this using a JSON reader and string reader and copypasta the sample response from the Civic Info API:
`representativesJson = Json.createReader(new StringReader("API response string")).readObject();`

https://docs.oracle.com/javaee/7/api/javax/json/Json.html
https://docs.oracle.com/javaee/7/api/javax/json/JsonReader.html
https://docs.oracle.com/javase/7/docs/api/java/io/StringReader.html

Reply:
@KathyFeiyang Might be worth rewriting your tests using this as well if you guys decide to go this route.

Reply:
Of course!

Reply:
Oops good catch using the gson library instead!